torch/onnx/_internal/fx/passes/fx_to_onnxscript.py

```diff
@@ -164,12 +195,16 @@ def _fill_tensor_meta(
 ):
     """Fill the meta information of onnxscript_values with that from the fx FakeTensor."""
 
+    # aten::sym_size output is a int, not a tensor, which stands
+    # for the size of one dim. We treat it as 1D tensor.
```

Comment:
1-D tensor or 0-D tensor? Why not 0-tensor?

Reply:
Done